tests/Integration/Behaviour/Features/Scenario/Cart/Calculation/RoundingType/round_total.feature

```diff
@@ -14,15 +14,15 @@ Feature: Cart calculation with rounding type TOTAL
     Given specific shop configuration for "rounding type" is set to round cart total
     Given there is a product in the catalog named "product1" with a price of 19.812 and 1000 items in stock
     When I add 1 items of product "product1" in my cart
-    Then my cart total should be precisely 26.812 tax included
+    Then my cart total should be precisely 26.81 tax included
     Then my cart total using previous calculation method should be precisely 26.812 tax included
 
   Scenario: one product in cart, quantity 3
     Given I have an empty default cart
     Given specific shop configuration for "rounding type" is set to round cart total
     Given there is a product in the catalog named "product1" with a price of 19.812 and 1000 items in stock
     When I add 3 items of product "product1" in my cart
-    Then my cart total should be precisely 66.436 tax included
+    Then my cart total should be precisely 66.44 tax included
```

Comment:
Also, the legacy method returns a more precise value, because it uses
`$compute_precision            = $this->configuration->get('_PS_PRICE_COMPUTE_PRECISION_');` as a precision
which is defined as 6 in the `config.inc.php` So I'm actually not sure using the same precision for display and computing is actually a good idea, it's not a revert it's drifting even further from the previous behaviour
We should return 6 to stay close to the initial behaviour

Reply:
if we force the computing precision to 6 we won't have a matching display and computing precision, so we'll have the same issues we're trying to fix in the first place...  or I didn't understand what you mean?

Reply:
Also the `_PS_PRICE_COMPUTE_PRECISION_` has been harcoded to 6 in 1.7.7, to match the results obtained in our behat tests... maybe we could do the same, and setting it to 2 ?